DNN Platform/Library/Common/Utilities/NetworkUtils.cs

```diff
@@ -242,6 +243,24 @@ public static bool IsIPInRange(string currentIP, string startIP, string subnetma
             }
             return false;
         }
+
+        /// <summary>
+        /// Get the Client IP Address.
```

Comment:
Since this is a new addition I'd like to get a bit more detail on this.  Something like

Gets the Client IP address of the current request, from server variables if available, otherwise returns Request.UserHostAddress